app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -245,7 +246,13 @@ protected void onResumeFragments() {
             return;
         }
 
-        tryAutoLogin();
+        if (CommCareApplication._().isConsumerApp()) {
+            uiController.setUsername("t1");
+            uiController.setPasswordOrPin("123");
+            localLoginOrPullAndLogin(false);

```

Comment:
Can you lift this into a helper method? Also, is there a reason for the `t1` / `123` values or is that something that is yet to be addressed?